### PR TITLE
Add more links that is corrected for apim 4.1.0

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Two key performance metrics were used to measure the performance of each test.
 
 ## Deployment used for the test
 
-Given below are the details of the deployment that was used for the performance test that is recorded in this documentation.
+deployment that was used for the performance test that is recorded in this documentation.
```
Comment (tharikaGitHub) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:36]: Is this change required?

Comment (chamindi-a) [reply en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md]: fixed [fa1c6d0](https://github.com/wso2/docs-apim/pull/7362/commits/fa1c6d04a76fe9ee8da64171b3245c7cd634f7e9)

**Review by tharikaGitHub — APPROVED** [review]
